packages/kit/src/views/Market/MarketDetailV2/components/TokenActivityOverview/utils/createTimeRangeOption.ts

```
@@ -0,0 +1,26 @@
+  | 'priceChange8hPercent'
+  | 'priceChange24hPercent';
+
+export function createTimeRangeOption(
+  tokenDetail: IMarketTokenDetail | undefined,
+  priceChangeKey: IPriceChangeKey,
+  label: string,
+  value: string,
+) {
+  const priceChangePercent = tokenDetail?.[priceChangeKey];
+  if (priceChangePercent) {
+    const percentage = parseFloat(priceChangePercent);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)